spec/development/workload/PageIterator.ts

```diff
@@ -0,0 +1,61 @@
+import { assert } from "chai";
+import { Event } from "microsoft-graph";
+
+import { PageIterator, PageIteratorCallback } from "../../../src/tasks/PageIterator";
+import { getClient } from "../test-helper";
+
+const client = getClient();
+describe("PageIterator", function() {
```

Comment:
the code change more requestoptions besides headers but those aren't tested.

Reply:
Added more tests